docs/_source/components-overview/pop_syn/binary_star.rst

```diff
@@ -22,98 +25,191 @@ BINARYPROPERTIES
+  * - ``step_times``
+    - Time spent in the steps in the evolution.
+  * - ``rl_relative_overflow_1``
+    - The relative overflow of the Roche Lobe of star 1.
+  * - ``rl_relative_overflow_2``
+    - The relative overflow of the Roche Lobe of star 2.
+  * - ``trap_radius``
+    - The trapping radius of the binary in R_sun.
+  * - ``acc_radius``
+    - The accretion radius of the binary in R_sun.
+  * - ``t_sync_rad_1``
+    - The synchronization time of the radiative zone of star 1 in yr.
+  * - ``t_sync_conv_1``
+    - The synchronization time of the convective zones of star 1 in yr.
+  * - ``t_sync_rad_2``
+    - The synchronization time of the radiative zone of star 2 in yr.
+  * - ``t_sync_conv_2``
+    - The synchronization time of the convective zones of star 2 in yr.
+  * - ``nearest_neighbour_distance``
+    - The distance to the nearest neighbour for NN interpolation
 
 
 Additional scalar properties can be added during the evolution.
 
 Since they do not change over time, they are not stored in the history.
-These can requested and will be stored in the output oneline (See the :ref:`Synthetic Populatiohn<synthetic-population>` and :ref:`Population Parameter Guide<pop-params-guide>` for more information).
+These can requested and will be stored in the output oneline (See the :ref:`Synthetic Population<synthetic-population>` and :ref:`Population Parameter Guide<pop-params-guide>` for more information).
 
-.. csv-table:: Additional columns
-  :header:  "Properties", "Descriptions"
-  :widths: 50, 150
-  'interp_class_HMS_HMS', "Description of interp_class_HMS_HMS."
-  'interp_class_CO_HMS_RLO', "Description of interp_class_CO_HMS_RLO."
-  'interp_class_CO_HeMS', "Description of interp_class_CO_HeMS."
-  'interp_class_CO_HeMS_RLO', "Description of interp_class_CO_HeMS_RLO."
-  'mt_history_HMS_HMS', "Description of mt_history_HMS_HMS."
-  'mt_history_CO_HMS_RLO', "Description of mt_history_CO_HMS_RLO."
-  'mt_history_CO_HeMS', "Description of mt_history_CO_HeMS."
-  'mt_history_CO_HeMS_RLO', "Description of mt_history_CO_HeMS_RLO."
+Additional columns
+~~~~~~~~~~~~~~~~~~
+
+The additional columns are defined as follows:
+
+.. list-table:: Additional columns
+  :header-rows: 1
+  :widths: 50 150
+
+  * - Properties
+    - Descriptions
+  * - ``interp_class_HMS_HMS``
+    - The interpolation class for the HMS_HMS phase.
+  * - ``interp_class_CO_HMS_RLO``
+    - The interpolation class for the CO_HMS_RLO phase.
+  * - ``interp_class_CO_HeMS``
+    - The interpolation class for the CO_HeMS phase.
+  * - ``interp_class_CO_HeMS_RLO``
+    - The interpolation class for the CO_HeMS_RLO phase.
+  * - ``mt_history_HMS_HMS``
+    -  The mass transfer history for the HMS_HMS phase.
+  * - ``mt_history_CO_HMS_RLO``
+    - The mass transfer history for the CO_HMS_RLO phase.
+  * - ``mt_history_CO_HeMS``
+    - The mass transfer history for the CO_HeMS phase.
+  * - ``mt_history_CO_HeMS_RLO``
+    - The mass transfer history for the CO_HeMS_RLO phase.
```

Comment:
I'm wondering, why we don't record stuff about the detached step and why we always have a column for each binary gird.
It might be more for a future PR (put in issue): this looks like a restriction to only go to each of the binary grid phases in maximum once per binary. I think, that takes away flexibility without any need.

Reply:
For `mt_history`, it makes sense to not record the detached step.
For `interp_class`, it might be useful